src/components/AnonymousChat/ChatDetailHeader.tsx

```diff
@@ -116,7 +116,7 @@ const ChatDetailHeader = ({
         </Text>
       </CustomPressable>
       <CustomPressable style={styles.threeDot} onPress={onThreeDotPress}>
-        <IconEP name="dots-three-vertical" size={20} color={COLORS.white} />
+        <IconEP name="dots-three-vertical" size={20} color={COLORS.white2} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the three-dot icon has been changed. As with previous color changes, ensure this is in line with the overall design and doesn't impact user experience negatively.
```diff
-    <IconEP name="dots-three-vertical" size={20} color={COLORS.white} />
+    <IconEP name="dots-three-vertical" size={20} color={COLORS.white2} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->